source/frontend/src/features/mapSideBar/property/MotiInventoryContainer.tsx

```diff
@@ -42,7 +43,13 @@ export const MotiInventoryContainer: React.FunctionComponent<
   const composedPropertyState = useComposedProperties({
     id: props.id,
     pid:
-      props?.pid === undefined || props?.pid === '' || isNaN(+props.pid) ? undefined : +props.pid,
+      props?.pid === undefined || props?.pid === '' || isNaN(Number(props.pid))
+        ? undefined
+        : Number(props.pid),
+    pin:
```

Comment:
would it make sense to use the pinparser here?

Reply:
yes, makes sense. Will make the change.